blenderproc/python/loader/HavenEnvironmentLoader.py

```diff
@@ -3,13 +3,15 @@
 import glob
 import os
 import random
+from mathutils import Euler
 
 import bpy
 
 from blenderproc.python.utility.Utility import Utility
 
 
-def set_world_background_hdr_img(path_to_hdr_file: str, strength: float = 1.0):
+def set_world_background_hdr_img(path_to_hdr_file: str, strength: float = 1.0, 
+                                 rotation_euler: Union[list, Euler, np.ndarray] = [0.0, 0.0, 0.0]):
     """
     Sets the world background to the given hdr_file.
```

Comment:
Please add documentation for the new parameter

Reply:
Added a documentation for the parameter `rotation_euler`.